### ISSUE TITLE
Adjusted phrases for the increasing friction alerts

### ISSUE THREAD

**Issue by terracoda** [phetsims/friction#119]
Moving from #106.

@zepumph, as I commented in (https://github.com/phetsims/friction/issues/106#issuecomment-420728699) I made a small change to the end of the increasing alerts and I think this should be beneficial in helping to create that gradual increase, though less gradual than the decrease as things just seem to happen faster in the increasing direction. 

I added changes in red in the design document, but basically things would go like this in terms of continuous warming (items 1 -6 already work well, so we just need items 7, 8, and 9):
1. On focus: "Atoms jiggle a tiny bit, temperature cool." (works well)
2. Start of friction: "Jiggling more, warmer." (works well)
3. Continue friction: "Jiggling faster, now hotter." (works well)
4. Continued increased friction: "Jiggling even faster, even hotter." (don’t see it as often, but seems to work well)
5. First evaporation point reached: "Very hot. Atoms break away from chemistry book." (works well)
6. Subsequent evaporation points until no more atoms available to evaporate: "Very hot. More atoms break away." (works well)

**What left to do are:**
- [ ] 7. NEW Single evaporation point alert when no atoms are left to evaporate: 
- "Very hot. Atoms jiggling very fast." 
- I think this replaces the current alert: “Jiggling very fast, still very hot.”

- [ ] 8. NEW Beyond evaporation point alert. I have 3 ideas:
a. "Jiggling even faster, now hotter." which is a nice comparative that is always true **OR**
b. "Jiggling super fast, temperature super hot." which reinforces the idea of a thermometer being present **OR**
c. "Jiggling super fast, super hot." which is short, sweet, and well super hot.

- [ ] 9. Lastly, I think an interaction hint to do a reset after two or three number 8, "beyond evaporation point" alerts, would be really helpful for users: 
- "Reset the sim to make more observations."

For the number 8 alert, I think I would like to try **8b** to reinforce the idea of thermometer.
I think number 8 alert would also replace the current alert of “Atoms very fast, still very hot,” but I am not 100% sure about that.

**Comment by zepumph** [comment]
After discussing with @terracoda, these are my notes on how to proceed (mainly the same as above)

7: add "Very hot. Atoms jiggling very fast." on break away, when no more atoms are left.


8: go with 8b, replace "Jiggling very fast, still very hot." with 8b (we can always change it later).


9: After three times of alerting 8b, replace 8b with a cue to reset the simulation, "Reset the sim to make more observations." This occurs until the sim is reset.

**Comment by terracoda** [comment]
These alerts all sound good. We may need to limit the number "Reset" hints in number 9 as it could interfere with doing repeated demos of jiggling/warming alerts. However, I am happy with the way it is implemented and I think the frequent "Reset" hints after all atoms have broken away with be very useful for non-visual users.

This is ready to close and I will open a new issue to discuss if there are too many "Reset" hints and mark it for a Friction design meeting.
